app/web/theme.ts

```diff
@@ -47,6 +47,9 @@ const themeOptions: ThemeOptions = {
           textDecoration: "none",
           color: "inherit",
         },
+        '.maplibregl-ctrl-bottom-right': {
+          bottom: '7px !important',
+        }
```

Comment:
I'm a little nervous about all these fixes mixed in here. Could we keep this PR to this feature and put the fixes in another PR?